docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -102,6 +103,22 @@ specify a browser alias that consists of the `{browser-provider-name}` prefix an
 testcafe "saucelabs:Chrome@52.0:Windows 8.1" tests/sample-fixture.js
 ```
 
+### Starting browser with arguments
+
+If you need to pass arguments for specified browser, you should write them after browser alias and surround this construction with quotation marks:
+
+```sh
+testcafe "chrome --start-fullscreen" tests/sample-fixture.js
+```
+
+You can also specify arguments for portable browsers. If the path to browser contains spaces, it should be surrounded with backticks:
```

Comment:
You can also specify arguments for portable browsers. If a path to a browser contains spaces, the path should be surrounded with backticks: